runtime/CMakeLists.txt

```diff
@@ -45,6 +45,18 @@ add_custom_command(OUTPUT ${GENERATED_HELP_TAGS}
   WORKING_DIRECTORY "${GENERATED_RUNTIME_DIR}/doc"
 )
 
+add_custom_command(OUTPUT ${GENERATED_HELP_TAGS}
+  DEPENDS
+    helptags
+)
+
+add_custom_command(OUTPUT doc_html
```

Comment:
Since I removed `doctags.c` I added this to be somewhat more explicit about what `make -C runtime/doc html` needs. Eventually that makefile should be eliminated but it's not worth it since we plan to move to a different html generator for the help docs anyways.